crypto/objects/obj_dat.c

```diff
@@ -670,7 +670,7 @@ int OBJ_create_objects(BIO *in)
             }
         } else
             s = NULL;
```

Comment:
How about adding {} to else according to "{} in both or neither". While you're at it...

Reply:
Well, there is another one at line 730. Well, given that it reduces chances for cherry-pick it either should be separate commit (if not request), or be ignored...